app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,23 @@
+
+<%= form_for(resource, as: resource_name, class:"signInForm", url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Username", class:"formInput"%>
+  </div>
+
+  <div class="field">
+    <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password", class:"formInput"%>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit( "Log in", class:"button") %>
+  </div>
+<% end %>
```

Comment:
- [ ] [Optional][Highly recommended] Please add a link that directs to the sign-up page, So when the dev server opens for the first time the user can go to the sign-up page from this link.